go/chat/boxer.go

```diff
@@ -846,6 +846,8 @@ func (b *Boxer) unboxV2orV3orV4(ctx context.Context, boxed chat1.MessageBoxed,
 	atMentions, atMentionUsernames, chanMention, channelNameMentions :=
 		b.getAtMentionInfo(ctx, clientHeader.Conv.Tlfid, membersType, body)
 
+	clientHeader.HasPairwiseMacs = len(boxed.ClientHeader.PairwiseMacs) > 0
```

Comment:
I'm talking about this, just return this len thing from the function you just created in extras. Can that not happen, what am I missing?

Reply:
we can't since `boxed.ClientHeader` is `MessageClientHeader` which has a `PairwiseMacs` field. We don't copy this over to `MessageClientHeaderVerified` on the unboxed message (besides this bool). I thought it would be better to just have this floating around instead all of the pairwise macs since we don't use them after we verify